extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -349,8 +360,7 @@ export abstract class AzureAuth implements vscode.Disposable {
 			return { accessToken, refreshToken, tokenClaims };
 
 		} catch (err) {
-			console.dir(err);
-			const msg = localize('azure.noToken', "Retrieving the token failed.");
+			const msg = localize('azure.noToken', "Retrieving the azure token failed. Please sign in again.");
```

Comment:
Capitalize Azure. Or maybe just call it something more generic like access token?